modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -2791,6 +2743,54 @@ public void testFollowerStartStopFollowing() throws Exception {
         cluster.ensureSame();
     }
 
+    @Test
+    public void testAppendEntriesWhenFollowerIsInErrorState() throws Exception {
```

Comment:
Why did you move this test?

Reply:
sorry, that was a mistake, reverted the test move